packages/graphile-build-pg/src/plugins/PgJWTPlugin.js

```diff
@@ -78,17 +78,18 @@ export default (function PgJWTPlugin(
                 pgJwtSecret,
                 Object.assign(
                   {},
-                  token.aud
+                  pgJwtSignOptions,
+                  token.aud || pgJwtSignOptions.audience
```

Comment:
`pgJwtSignOptions` may not be provided, which will cause this code to error. Using a default in the destructuring won't work if the setting is set to `null`, so I recommend adopting this approach:

```suggestion
                  token.aud || pgJwtSignOptions && pgJwtSignOptions.audience
```

Reply:
(And similar for the other entries.)

Reply:
Done